src/main/resources/com/google/api/codegen/java/main.snip

```diff
@@ -380,27 +405,64 @@
 @end
 
 @private callableMethodImpl(apiMethod)
-  @if apiMethod.isStreaming
-    {@streamingCallableMethod(apiMethod)}
-  @else
-    {@nonStreamingCallableMethod(apiMethod)}
-  @end
+  public final UnaryCallable<{@apiMethod.serviceRequestTypeName}, \
+      {@apiMethod.responseTypeName}> {@apiMethod.name}() {
+    return {@apiMethod.callableMethod.callableName};
+  }
 @end
 
-@private nonStreamingCallableMethod(apiMethod)
-  public final UnaryCallable<{@apiMethod.apiRequestTypeName}, \
+@private streamingCallableMethod(apiMethod)
+  @let coreSampleCode = streamingCallableMethodSampleCode(apiMethod)
```

Comment:
We have stopped using let statements in most places - is it necessary here, for formatting or parsing reasons? If not, would a line break also work?